pyproject.toml

```diff
@@ -1,35 +1,671 @@
+[project]
+name = "databricks-labs-discoverx"
+dynamic = ["version"]
+description = 'Common libraries for Databricks Labs'
```

Comment:
Add discoverx description, not the generic one

Reply:
Updated